truss/templates/triton/model/1/model_wrapper.py

```diff
@@ -0,0 +1,70 @@
+import importlib
+import inspect
+import sys
+from pathlib import Path
+from typing import Dict
+
+from shared.secrets_resolver import SecretsResolver
+from triton_utils import (
+    _inspect_pydantic_model,
+    _signature_accepts_keyword_arg,
+    _transform_pydantic_type_to_triton,
+    _transform_triton_requests_to_pydantic_type,
+)
+
+
+class ModelWrapper:
```

Comment:
Why are we making a copy of this class here? Is there anyway to refactor the existing model_warpper to handle this so the code is not duplicated?